src/hardware/sblaster.cpp

```diff
@@ -2196,7 +2221,8 @@ class SBLASTER final {
 	IO_WriteHandleObject write_handlers[0x10] = {};
 
 	static constexpr auto blaster_env_name = "BLASTER";
-	OplMode oplmode = OplMode::None;
+	OplMode oplmode                        = OplMode::None;
```

Comment:
Minor: Maybe insert a blank line above this, so much blank space looks ridiculous 😄